src/components/dropzone/index.md

```diff
@@ -473,14 +473,27 @@ If you want reset the value when canceling, use `clearOnCancel` prop to reset th
 | `maxlength`    |      `Number`       |   `-`   | Maxlength selected files, only work on multiple mode                           |
 | `accept`       |      `String`       |   `-`   | Limitting file type can be selected, see [HTML attribute: accept][attr-accept] |
 | `clearOnCanel` |      `Boolean`      | `false` | Clear v-model when user cancel the browse file                                 |
-| `modelValue`   |       `File`        |   `-`   | v-model value                                                                  |
+| `v-model`   |       `File`        |   `-`   | v-model value                                                                  |
```

Comment:
@afrijaldz The prop name was correct `modelValue`, there no prop named `v-model`